ArcadiaParties-SPA/src/app/user/user.module.ts

```diff
@@ -0,0 +1,28 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterModule } from '@angular/router';
+import {StoreModule} from '@ngrx/store';
+
+import { UserIdentityComponent } from './user-identity/user-identity.component';
+import { UserInfoComponent } from './user-info/user-info.component';
+import { MaterialModule } from '../material/material.module';
+import { appRoutes } from '../routes';
+import { userFeatureStateKey, userReducer } from './reducers/user-reducer';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    RouterModule.forRoot(appRoutes),
```

Comment:
Root routes are registered only once. If you need to register additional routes in feature module, you should use `forChild`. But you don't need it, because you don't add any new routes.

Reply:
`forChild` added